lineman/app/components/navbar/navbar_user_options.haml

```diff
@@ -2,7 +2,9 @@
   .blank{uib-dropdown: ''}
     %button.btn.lmo-btn-nude.lmo-navbar__btn.lmo-navbar__btn-icon{href: '', tabindex: 5, uib-dropdown-toggle: ''}
       .sr-only{translate: 'navbar.user_options.label'}
-      %user_avatar{user: 'currentUser', size: 'small'}
+      %user_avatar{user: 'currentUser', size: 'small', ng-if: 'currentUser.avatarKind != "initials"'}
+      .navbar-user-options__user-profile-icon.user-avatar
```

Comment:
Doesn't this want to be an `ng-if: 'currentUser.avatarKind == "initials"`? Otherwise it's still visible but lurking under the user_avatar... seems potentially troublesome.